### PR TITLE
Don't panic in build.rs

### PR DESCRIPTION
**Pull request by tokatoka**
It's a huge PAIN that this flaky guy fails everytime in vscode

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ members = [
     "libafl_derive",
     "libafl_frida",
     "libafl_libfuzzer",
-    "libafl_nyx",
+    # "libafl_nyx",
```
Comment (domenukk) [inline Cargo.toml:14]: Don't comment it out though

File: libafl_nyx/build.rs [hunk 2]
```diff
@@ -7,10 +7,8 @@ fn main() {
         let status = Command::new("./build_nyx_support.sh")
             .status()
             .expect("can't run ./build_nyx_support.sh");
-        if status.success() {
-            println!("success to run ./build_nyx_support.sh");
-        } else {
-            panic!("fail to run ./build_nyx_support.sh");
+        if !status.success() {
+            println!("cargo:warning=fail to run ./build_nyx_support.sh");
```
Comment (domenukk) [inline libafl_nyx/build.rs:11]: Should we instead ... fix `build_nyx_support`?

Comment (domenukk) [reply libafl_nyx/build.rs:11]: Or.. should that even be in build.rs or instead in a makefile?

Comment (tokatoka) [reply libafl_nyx/build.rs:11]: it is not necessarily a problem in that shell script
because sometimes it is network issue
sometimes it is git issue

**Comment by tokatoka** [comment]
i'll move others out from members too and change ci

**Comment by tokatoka** [comment]
in #2391

**Pull request closed without merging**